dns/setup.py

```diff
@@ -22,7 +22,7 @@
 
 name = 'google-cloud-dns'
 description = 'Google Cloud DNS API client library'
-version = '0.30.1'
+version = '0.30.2'
```

Comment:
I wonder if the endpoint change is worth signalling with a minor release?

Reply:
I went back to ask folks doing this in other languages what they're doing Will leave it open for now.